tfhe/src/integer/server_key/radix_parallel/tests.rs

```diff
@@ -716,6 +717,39 @@ where
     }
 }
 
+fn integer_default_bitnot<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let (cks, sks) = KEY_CACHE.get_from_params(param);
+    let cks = RadixClientKey::from((cks, NB_CTXT));
+
+    //RNG
+    let mut rng = rand::thread_rng();
+
+    // message_modulus^vec_length
+    let modulus = cks.parameters().message_modulus().0.pow(NB_CTXT as u32) as u64;
+
+    for _ in 0..NB_TEST {
+        // Define the cleartexts
+        let clear = rng.gen::<u64>() % modulus;
+
+        // Encrypt the integers
+        let ctxt = cks.encrypt(clear);
+
+        // Negates the ctxt
+        let ct_tmp = sks.bitnot_parallelized(&ctxt);
```

Comment:
Missing determinism check for default ops, check how other default tests are written